testdata/vast4_universal_ad_id.xml

```diff
@@ -1,5 +1,5 @@
-<VAST version="4.0" 
-  xmlns:xs="http://www.w3.org/2001/XMLSchema" 
+<VAST version="4.2"
```

Comment:
shouldn't we rather copy and duplicate the testdata for 4.2? Or does it really needs changing?

Reply:
hmm I think we can leave it as it is in (4.0) in this test data but for the other two datasets I'd leave 4.2 cause they contain features introduced by 4.2